### PR TITLE
Glt 2091 Pinery-MISO tests

### PR DESCRIPTION
**Pull request by djcooke**
GLT-2079 commit is in separate PR #195, which should be merged first

### CODE REVIEW

**Comment by djcooke** [comment]
Waiting for #195 to be merged before I rebase this branch to fix the build

File: pinery-miso/src/main/resources/ca/on/oicr/pinery/lims/miso/queryAllSamples.sql [hunk 1]
```diff
@@ -0,0 +1,285 @@
+SELECT s.alias NAME
+        ,s.description description
+        ,s.NAME id
+        ,parent.NAME parentId
+        ,COALESCE(tt.sampleTypeName, sc.alias) sampleType
```
Comment (callunity) [inline pinery-miso/src/main/resources/ca/on/oicr/pinery/lims/miso/queryAllSamples.sql]: How does this line work? My understanding is that it involves searching for TissueType.sampleTypeName column, but that doesn't exist in the database.

Comment (djcooke) [reply pinery-miso/src/main/resources/ca/on/oicr/pinery/lims/miso/queryAllSamples.sql]: `TissueType.sampleTypeName` does exist in OICR DB. Was added to preserve the old "sample type" in Pinery when we reduced the Tissue classes down to just "Tissue"

**Review by callunity — APPROVED** [review]

File: pinery-miso/src/main/resources/ca/on/oicr/pinery/lims/miso/queryAllSamples.sql (hunk 1 above)
Comment (callunity) [reply pinery-miso/src/main/resources/ca/on/oicr/pinery/lims/miso/queryAllSamples.sql]: Aha, I'd forgotten that was there.

File: pinery-miso/pom.xml [hunk 2]
```diff
@@ -0,0 +1,268 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>uk.ac.bbsrc.tgac.miso</groupId>
+    <artifactId>miso</artifactId>
+    <version>0.2.94-oicr-SNAPSHOT</version>
```
Comment (callunity) [inline pinery-miso/pom.xml]: The Travis error might be fixed by updating this to 0.2.97-oicr-SNAPSHOT

**Review by tonydebat — APPROVED** [review]

**Pull request merged**